v1/content.yaml

```diff
@@ -180,7 +180,11 @@ paths:
 
   /html/{title}:
     x-route-filters:
-      - path: ./lib/revision_table_access_check_filter.js
+      #- path: ./lib/revision_table_access_check_filter.js
+      #  options:
+      #    redirect_cache_control: '{{options.purged_cache_control}}'
+      #    attach_body_to_redirect: true
+      - path: ./lib/access_check_filter.js
```

Comment:
Why are we updating only these 2 endpoints? I thought we would replace the filter everywhere.